inst/azure/azure_storage_setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
```

Comment:
`inst/` is better than `R/` for shell scripts, but I still have the concerns above.